game_patch/sound/sound.cpp

```diff
@@ -772,7 +772,7 @@ void play_chat_sound(std::string& chat_message, bool is_taunt)
     }
 }
 
-void snd_ds_apply_patch();
+extern void snd_ds_apply_patch();
```

Comment:
Why? Not really an issue, I just don't know that this does _anything_ here at all and I'm confused as to why its in this PR?